WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerCategoryCollectionViewCell.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+class LayoutPickerFilterCollectionViewCell: UICollectionViewCell {
+
+    static var cellReuseIdentifier: String {
+         return "LayoutPickerCategoryCollectionViewCell"
+     }
+
+    static var nib: UINib {
+        return UINib(nibName: "LayoutPickerCategoryCollectionViewCell", bundle: Bundle.main)
+    }
```

Comment:
Again here I wanted to mention converting these to constants, but it's just a nitpick.